eventservice/api/controllers/events.py

```diff
@@ -213,7 +213,7 @@ def _get_event_result(query):
 
 
 def post():
-    auth_middleware.authorize(auth_middleware.rokwire_event_manager_group)
+    auth_middleware.authorize(auth_middleware.ROKWIRE_EVENT_WRITE_GROUPS)
```

Comment:
@bingzhang, this needs to be applied for all POST/PUT/PATCH/DELETE methods for images as well. The rest looks good. Thanks.